README.md

```diff
@@ -1,17 +1,82 @@
-# Welcome 👋
+# GitHub Working Draft
 
+# Intro
 
-Thank you for using Storyblok and for taking the time to help us make it better! If you have found a bug or have a suggestion for a new feature, we would love to hear from you. 
+Storyblok is a visual CMS that works with any framework. Integrate with any app or frontend or framework. Publish anywhere with an API-first backend. Storyblok’s industry-leading Visual Editor makes it easy for marketing teams to independently create, edit, and collaborate.  
 
-## Looking for the Storyblok CLI?
+# Getting Started
 
-The Storyblok CLI is now in it's own separate repository. You can find it here: <https://github.com/storyblok/storyblok-cli>
+- [https://www.storyblok.com/docs/guide/getting-started](https://www.storyblok.com/docs/guide/getting-started) → this should be replaced by the new Quickstart page once it’s live
```

Comment:
Please change the link title to "Get started with your favorite technology".
And remove the internal note ("this should be replaced...").